.github/workflows/create-daily-docs-sync-pr.yaml

```diff
@@ -0,0 +1,69 @@
+name: daily docs sync PR
+
+on:
+  schedule:
+    # Runs "at minute 44 past midnight" (see https://crontab.guru)
+    # Reason is avoiding full hour peak
+    - cron: "44 0 * * *"
+
+env:
+  BOT_USERNAME: bot_name
+  BOT_EMAIL: bot_mail@example.com
+  GITHUB_REPOSITORY_OWNER: solidity-docs
+  # comma separated list of labels
+  LABELS: sync,automated-pr
+
+jobs:
+  createPullRequest:
+    runs-on: ubuntu-latest
+    strategy:
+      # In the context matrix strategy, fail-fast means, if one of the jobs
+      # fails,the rest of the jobs will be canceled. In our case, this can
+      # be a limitation. When merging or creating PR to one repository fails,
+      # the rest still can be good.
+      fail-fast: false
+      matrix:
+        # This means, that all pull requests will be processed simultaneously
+        # and independently of each other.
+        repos: ["de-german", "fr-french"]
```

Comment:
Can you make this empty for now? Or, if it can't be empty, just disable the bot by commenting out the schedule? We want to test the bot first and leaving this here would make it already run in these repos.

Reply:
an array can not be empty. replaced with `repos: ["placeholder"]`

Reply:
Let's just disable the schedule then. You can do it like this:
```yaml
on:
  push:
    # Action disabled. The filter excludes all the possible branch names.
    branches: ["!*"]
```

If the array can't be empty, I'd rather keep an actual language name there.